src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -348,7 +348,7 @@ public enum CassandraRelevantProperties
     MEMTABLE_OVERHEAD_SIZE("cassandra.memtable.row_overhead_size", "-1"),
     MEMTABLE_SHARD_COUNT("cassandra.memtable.shard.count"),
     MEMTABLE_TRIE_SIZE_LIMIT("cassandra.trie_size_limit_mb"),
-    MIGRATION_DELAY("cassandra.migration_delay_ms", "60000"),
```

Comment:
Shouldn't this be deprecated before removal?

Reply:
it has no effect anymore, and for users setting it cassandra will still start without issue

Reply:
I am with you on this; it was just something people do in the community for one major release, even if the property has no effect. But on second thought, that makes more sense for the Config class than here. In this class, the Deprecated parameters can still be used. I just checked them.